src/main/java/oss/fosslight/service/impl/HistoryServiceImpl.java

```diff
@@ -22,61 +22,61 @@
 @Service
 public class HistoryServiceImpl extends CoTopComponent implements HistoryService {
 	@Autowired HistoryMapper hisotryMapper;
-	
+

```

Comment:
Lets' do not remove spaces that aren't relevant to your edits.

Reply:
In the future, if irrelevant modifications (ex. file indentation modification) are needed, it is recommended to separate the commits.